src/rgw/driver/rados/rgw_sal_rados.cc

```diff
@@ -3428,9 +3428,9 @@ int RadosObject::RadosDeleteOp::delete_obj(const DoutPrefixProvider* dpp, option
   parent_op.params.abortmp = params.abortmp;
   parent_op.params.parts_accounted_size = params.parts_accounted_size;
   parent_op.params.null_verid = params.null_verid;
-  if (params.objv_tracker) {
-      parent_op.params.check_objv = params.objv_tracker->version_for_check();
-  }
+  /* don't assume zero-initialization of pointers */
```

Comment:
shouldn't we _always_ assume that pointer members are zero-initialized?

i'm not spotting the problem here - `RGWRados::Object::DeleteParams::check_objv` is initialized by its ctor